src/Microsoft.Framework.Runtime.Hosting/Json/JsonDeserializer.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Framework.Runtime.Json
+{
+    internal class JsonDeserializer
```

Comment:
Can this class just be a `static` class? It has no meaningful state and it doesn't seem like an instance of this type can be meaningfully reused. And, all current usage of this type is single use anyway. Recommend making it static.